rca/project_styleguide/templates/patterns/molecules/key-details/key-details--project.html

```diff
@@ -27,10 +50,10 @@ <h4 class="body body--two key-details__sub-heading">RCA expertise</h4>
             </ul>
         </div>
     {% endif %}
-    {% if page.key_details_download %}
+    {% if page.specification_document %}
         <div class="key-details__section key-details__section--action">
-            <a class="key-details__link link link--secondary link--download" href="{{ page.page.key_details_download }}">
-                <span class="link__label">Download the full {% if page.key_details_download_course %}{{ page.key_details_download_course }} {% endif %}programme specification</span>
+            <a class="key-details__link link link--secondary link--download" href="{{ page.specification_document }}">
+                <span class="link__label">Download the full {% if page.specification_document %}{{ page.specification_document }} {% endif %}programme specification</span>
```

Comment:
Is the repeated `{% if page.specification_document %}` needed?

Reply:
probably not - will remove